Junction/example-normalized-ld.jsonld

```diff
@@ -1,70 +1,378 @@
 {
-    "id": "urn:ngsi-ld:Junction:63fe7d79-0d4c-4da9-b7d0-3340efa0656a",
-    "type": "Junction",
-    "createdAt": "2020-02-20T15:42:00Z",
-    "modifiedAt": "2020-02-20T15:45:00Z",
-    "location": {
-        "type": "GeoProperty",
-        "value": {
-            "type": "Point",
-            "coordinates": [
-               24.30623,
-               60.07966
+    "$schema": "http://json-schema.org/schema#",
```

Comment:
this file should contain an NGSI-LD example, not the JSON schema

Reply:
Done :D